packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/components/Controls/components/PeriodMultipleControls/PeriodMultipleControls.spec.js

```diff
@@ -0,0 +1,80 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+import sinon from 'sinon'
+import { SpacedText } from '@zooniverse/react-components'
+import { FormField } from 'grommet'
+import { PeriodMultipleControls, StyledRadioButtonGroup } from './PeriodMultipleControls'
+import en from '../../../../locales/en'
+
+const options = ['0.5', '1', '2', '3'].map((option) => {
+  return {
+    label: <SpacedText size='10px' style={{ width: '1ch' }} weight='bold'>{option}</SpacedText>,
+    value: option
+  }
+})
+
+describe('Controls > Components > PeriodMultipleControls', function () {
+  it('should render without crashing', function () {
+    const wrapper = shallow(
+      <PeriodMultipleControls
+        options={options}
+        periodMultiple={1}
+        setPeriodMultiple={() => {}}
+      />
+    )
+    expect(wrapper).to.be.ok()
+  })
+
+  describe('FormField', function () {
+    let wrapper, formField
+    before(function () {
+      wrapper = shallow(
+        <PeriodMultipleControls
+          options={options}
+          periodMultiple={1}
+          setPeriodMultiple={() => { }}
+        />
```

Comment:
wrapper is set three times in this test. The only difference being that on one occasion, `setPeriodMultiple` has a spy attached. Could they all be defined the same way to utilize a `beforeEach` and setting the wrapper once?